assemble/conf/log4j-monitor.properties

```diff
@@ -15,18 +15,18 @@
 
 # Write out everything at the DEBUG level to the debug log
 log4j.appender.A2=org.apache.log4j.RollingFileAppender
-log4j.appender.A2.File=${org.apache.accumulo.core.dir.log}/${org.apache.accumulo.core.application}_${org.apache.accumulo.core.ip.localhost.hostname}_fromprops.debug.log
+log4j.appender.A2.File=${accumulo.log.dir}/${accumulo.application}_${accumulo.local.hostname}.debug.log
```

Comment:
Where are all these properties being set? Are we to assume they will always be set in `accumulo-env.sh` (rather than being set by us in code)? If that's the case (and I hope it is... I don't want to start publishing system properties as public API), we should have a comment about where these properties come from, so that the user will understand that it's their responsibility to ensure they are set in their environment for this configuration to work.

I also think this could be simpler... rather than having a huge concatenation of many system properties, all of which need to be set in the environment file, it'd be better to have a single one which is set in the environment file. It's easier to document.

Example:

In log4j config: `log4j.appender.A2.File=${accumulo.debug.log.filename}`
In accumulo-env.sh: `JAVA_OPTS=(... "-Daccumulo.debug.log.filename=$file")`